### ISSUE TITLE
sofk estimator test uses python pandas

### ISSUE THREAD

**Issue by prckent** [QMCPACK/qmcpack#1273]
#764 test appears to use pandas. We have to be very cautious about python dependencies and make sure the benefits are worthwhile. We should have a formal list in the documentation and README.md. For tests I would prefer the minimum number of tests since it increases the likelihood that the tests will actually be run and used. Eyeballing check_properties_h5dat.py it is for the dataframe functionality only.  Can we avoid it?

**Comment by jtkrogel** [comment]
Yes we can.  The scalar.dat file is pretty easy to read, e.g. with numpy loadtxt.
